lib/rules/no-sync.js

```diff
@@ -13,6 +13,10 @@
 
 module.exports = {
     meta: {
+        deprecated: true,
+
+        replacedBy: ["https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-sync.md"],
```

Comment:
Should this be the rule name instead of the URL?

Reply:
I was wondering that too, and ended up leaning toward this to save folks the step of having to google it. That being said, it does feel weird. Happy to go with consensus on this.

Reply:
It looks like we don’t have other examples using URLs in this spot, so it seems using the rule name would be consistent.